GOVERNANCE.md

```diff
@@ -65,6 +65,24 @@ meeting. In both cases the decision must be seconded by a majority of the Team
 or else the discussion will continue. Simple majority wins. Only Active
 Members participate in a vote.
 
+## Aproving Project Charters
+
+As per the CPC charter the voting CPC members are responsible for approving project
+charters and changes to them as per section
+[5](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#section-5-responsibilities-and-expectations-of-the-cpc).
+
+Approval of the initial charter or changes to an existing charter will
+be approved by opening an issue requesting approvle in the cross-project-council
+[repository](https://github.com/openjs-foundation/cross-project-council/)
+
+The request will be approved through our standard concensus seeking process with the following
+conditions:
+
+* there are no outstanding objections
+* there are two or more approvals by `voting` CPC members
```

Comment:
It has been discussed that maybe this needs to be approved by a quorum of voting members instead?  @openjs-foundation/cpc please chime in.

Reply:
Without refering to the specifics of this case, my gut instinct is there should be as few permutations as possible for how things are approved. That will remove confusion of what is required for any particular decision as they come up.

'Approval by a quorum of CPC voting members' is the standard baseline of approval and I don't necessarily see why this would be different.

Reply:
```suggestion
* there are two or more approvals by voting CPC members
```

Reply:
I agree that a quorum of voting members is the goal, but I think this is meant to address how that approval carries over to GitHub. Specifically, forcing a quorum of GitHub approvals on all PRs could get tedious in practice and end up with lots of "please add approval so we can merge" comments. Alongside the other requirements (being open 14 days, having no objections, and the board having been consulted for substantive issues), 2 explicit GitHub approvals is perhaps enough to assume a quorum.